Dockerfile.prow

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+# Build the manager binary
+FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux-amd64 AS builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+
+# Build
+RUN CGO_ENABLED=0 go build -a -o manager main.go
```

Comment:
Should we add `GOGC=25`? We don't usually have memory problems on this repo -- just for consistency

Reply:
Let's keep it simple. We only see problems in the aggregator project and it seems to be better with newer versions of go and libraries, so maybe we can eventually remove from all repos.